helm/zeebe-test/values.yaml

```diff
@@ -371,57 +367,171 @@ ph-ee-engine:
     elasticsearch_sslverification: false
     elasticsearch_security_enabled: false
     reporting:
-      enabled: false
+      enabled: true
       fields:
         amount: true
         accountId: true
-        errorCode: false
+        errorCode: true
         errorDescription: true
         externalId: true
-        initiator: false
-        initiatorType: false
-        isNotificationsFailureEnabled: false
-        isNotificationsSuccessEnabled: false
-        mpesaTransactionId: false
-        mpesaTransactionStatusRetryCount: false
-        originDate: false
-        partyLookupFailed: false
+        initiator: true
```

Comment:
We need not have this many reporting variables enabled in a chart not meant for load testing as this consumes a non-trivial amount of resources.